src/main/webapp/index.html

```diff
@@ -19,23 +19,30 @@
   <head>
     <meta charset="UTF-8">
     <title>CodeU 2019 Starter Project</title>
-    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" type="text/css" href="/css/main.css">
+    <script type="text/javascript" src='js/main.js'></script>
     <script src="/js/navigation-loader.js"></script>
   </head>
   <body onload="addLoginOrLogoutLinkToNavigation();">
-    <nav>
-      <ul id="navigation">
-        <li><a href="/">Home</a></li>
-        <li><a href="/aboutus.html">About Our Team</a></li>
-        <li><a href="/feed.html">Public Feed Page</a></li>
-        <li><a href="/community.html">Community Page</a></li>
-      </ul>
-    </nav>
-    <h1>CodeU Starter Project</h1>
-    <p>This is the CodeU starter project. Click the links above to login and visit your page.
-    You can post messages on your page, and you can visit other user pages if you have
-    their URL.</p>
-    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!
-    You could start by modifying this page to tell the world more about your team.</p>
+    <header>
+      <img src="images/codeUIcon.png" alt="Image not available">
+      <nav>
+        <ul>
+          <li><a href="/">Home</a></li>
+          <li><a href="/aboutus.html">About Our Team</a></li>
+          <li><a href="/feed.html">Public Feed Page</a></li>
+          <li><a href="/community.html">Community Page</a></li>
+          <li><a href="/maps.html">Maps Page</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h1>CodeU Starter Project</h1>
```

Comment:
I think this section can be removed.

Reply:
![image](https://user-images.githubusercontent.com/26247692/60089663-90547c80-9773-11e9-8fde-f0807af812a1.png)